app/clinic/views/patient-state.html

```diff
@@ -1,12 +1,16 @@
 <div class="row bg-info-border content-border border-rouded"><!--patient header -->
-    <div class="col-md-4 well well-sm panel-marginless-bottom">
-        <p>CD4: <span class="label label-danger">300</span> desde: 2016-5-14</p>
-        <p>Carga Viral: <span class="label label-primary">200</span> desde: 2016-5-14</p>
-        <p>Estadio OMS: <span class="label label-primary">II</span> desde: 2016-5-14</p>
-    </div>
+
+  <div class="col-md-4 well well-sm panel-marginless-bottom" ng-repeat="patientState in patientStates">
+    <p ng-class="capitalize"> {{patientState.exam}} :
+      <span class="label label-danger">{{patientState.result}}</span> desde: {{labDate}}</p>
```

Comment:
Notice that the label color should be dynamic, based on the severity of the result value for e.g. `CD4 count < 500` should display red or blue otherwise.